syntaxhighlighter.php

```diff
@@ -11,7 +11,7 @@
 Text Domain:  syntaxhighlighter
 License:      GPL2
 License URI:  https://www.gnu.org/licenses/gpl-2.0.html
-Requires at least: 5.7
+Requires at least: 6.1
```

Comment:
This aligns the minimum WP version supported with the one supported by WPJM.

Given that 6.1 is already ~1 year old, and 5.7 is almost ~3 (!!!) years old to this point, I think it is safe to ask people to upgrade. :)

Reply:
If it works with 5.7, let's keep it that way (if it doesn't, then, fine :D).
We won't necessarily make any breaking change in the near (or far) future, so if this release doesn't break compat with 5.7, we should keep it as-is.

Reply:
Fair enough, I tested it on PHP 7.0 and WP 5.7 and reverted the commit in 9f582dc. Thanks!